stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+labels:
+  io.cattle.role: project # options are cluster/project
+questions:
+# Global Options
+- variable: harborAdminPassword
+  required: true
+  description: "The initial password of Harbor admin. Change it from portal after launching Harbor"
+  type: password
+  label: The Initial Password of Harbor Admin
+  group: "Global Options"
+- variable: persistence.enabled
+  default: true
+  required: true
+  description: "Enable persistent storage for Harbor components"
+  type: boolean
+  label: Enable Persistent Storage
+  group: "Global Options"
+# Harbor Expose Service
+- variable: expose.type
+  required: true
+  default: "ingress"
+  description: "Set the way how to expose the service"
+  type: enum
+  label: Harbor Proxy Service Type
+  group: "Expose Service Options"
+  options:
+  - "ingress"
+  - "loadBalancer"
+  - "nodePort"
+  - "clusterIP"
+- variable: expose.ingress.hosts.core
+  required: true
+  default: "harbor.domain"
+  description: "The host of Harbor core service in ingress rule, e.g, harbor.domain"
+  type: string
+  label: Ingress Hostname of Harbor
+  group: "Expose Service Options"
+  show_if: "expose.type=ingress"
+- variable: expose.ingress.hosts.notary
+  required: true
+  default: "notary.domain"
+  description: "The host of Harbor notary service in ingress rule, e.g, notary.domain"
+  type: string
+  label: Ingress Hostname of Harbor Notary
+  group: "Expose Service Options"
+  show_if: "expose.type=ingress"
+- variable: externalURL
+  default: "https://harbor.domain"
+  description: "Set the external URL used to access Harbor. If exposed using an Ingress, it should reflect the Ingress Hostname used for Harbor (format: protocol://domain:port)"
+  type: string
+  label: The External URL for Harbor Service
+  group: "Expose Service Options"
+- variable: expose.loadBalancer.IP
+  default: ""
+  description: "Set the IP if the LoadBalancer supports assigning IP"
+  type: string
+  label: Set LoadBalancer IP
+  show_if: "expose.type=loadBalancer"
+  group: "Expose Service Options"
+- variable: expose.tls.certSource
+  required: true
+  default: "auto"
+  description: "The source for the Harbor TLS certificate"
```

Comment:
What about adding a line that "auto" means auto generation of certs?

Reply:
I extended the description to explain what those values are.